CHANGES.md

```diff
@@ -8,7 +8,7 @@ Release Notes.
 * Chore: adapt `create_source_release.sh` to make it runnable on Linux.
 
 #### Java Agent
-
+* Update operationName of quartz-scheduler plugin.
```

Comment:
Is this description and PR title correct? The codes seem to add a new tag only. Please confirm.

Reply:
updated [operationName](https://github.com/apache/skywalking/pull/5934/files#diff-818769a4ff74a286eae8711cb67b36f71145fca4d6ebbcaed32d16d0a93181b7R54)

Reply:
Got it.